staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap.go

```diff
@@ -86,8 +86,8 @@ type ConfigMapOptions struct {
 	FileSources []string
 	// LiteralSources to derive the configMap from (optional)
 	LiteralSources []string
-	// EnvFileSource to derive the configMap from (optional)
-	EnvFileSource string
```

Comment:
This struct and field are exported. This will be a breaking change to any external consumers. Not sure if there are any 🤔.

https://cs.k8s.io/?q=EnvFileSource&i=nope&files=&excludeFiles=&repos=

Reply:
I guess with our refactoring of the commands anyways it doesn't matter.

Reply:
My instinct is that the flags used to build a CLI are particular to that CLI itself, and not intended for library use. That search isn't really showing any consumers in any case; Kustomize's implementation matches but is independent, and driver-registrar is just showing up as vendored, not used. The rest is this code itself and its mirror. So IMO there's no problem with this change, but we can check with other approvers if you're worried about it. :)